features/step_definitions/helper_services.rb

```diff
@@ -201,7 +201,7 @@
   raise "cannot create git-server service" unless @result[:success]
 
   # wait to become available
-  @result = BushSlicer::Pod.wait_for_labeled("deployment-config=git-server",
+  @result = BushSlicer::Pod.wait_for_labeled("deploymentconfig=git-server",
```

Comment:
Does the change works on all 4.x ? E.g, do we test it on 4.2 and 4.5 ?

Reply:
@liangxia  Yes passed in[ 4.2](https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/108483/console)  and [4.5](https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/108501/console)

Reply:
@wewang58  Thank you for the confirm.